asciidoc/asciidoc.py

```diff
@@ -1872,7 +1939,21 @@ def getnumber(level):
         return number
 
 
-class FloatingTitle(Title):
+class FloatingTitleMeta(TitleMeta):
```

Comment:
I don't think this is correct, as we want `FloatingTitleMeta` to inherit the various properties/methods of the base `Title` class...